wss-unified-agent.config

```diff
@@ -0,0 +1,20 @@
+# WhiteSource documentation https://whitesource.atlassian.net/wiki/spaces/WD/pages/1544880156/Unified+Agent+Configuration+Parameters
+
+excludes=ruling/**
+fileSystemScan=False
+resolveAllDependencies=False
+
+maven.aggregateModules=True
+maven.downloadMissingDependencies=False
+maven.m2RepositoryPath=.m2/repository
+maven.resolveDependencies=True
+maven.runPreStep=False
+
+npm.includeDevDependencies=True
+npm.resolveDependencies=True
```

Comment:
Just to be sure: We actually want the dependency resolution to use the lock file (default behavior)? In the SonarJS configuration,  we explicitly added `npm.resolveLockFile=False`.

Reply:
yes, I think it's better actually. I will change it also in sonarjs